themes/mcf/tailwind.config.js

```diff
@@ -89,6 +92,9 @@ module.exports = {
       },
       letterSpacing: {
         slight: "0.015em",
+        normalHeading: "-0.01em",
+        xtraLargeHeading: "-0.04em",
+        subheading: "0.04em",
```

Comment:
Nice!

Reply:
Combine these new settings with the `Heading` mentioned in the last comment below